### PR TITLE
Watch mode

### PR DESCRIPTION
**Pull request by alatarum**
In watch mode lssensors would print sensor values every <n> seconds.
This mode can be useful to see system dynamics and prepare data for
graph drawing.

### CODE REVIEW

File: list-sensors.cpp [hunk 1]
```diff
@@ -354,6 +376,83 @@ struct CmpSensorsName
...
+        bool found = false;
+        for (const auto& obj : objects)
+        {
+            size_t name_pos = obj.first.rfind('/');
+            if (name == obj.first.substr(name_pos + 1))
+            {
+                found = true;
+                for (const auto& service : obj.second)
+                {
+                    sensors.emplace_back(std::pair(service.first, obj.first));
```
Comment (nest1ing) [inline list-sensors.cpp:413]: `std::pair` is excess here, `emplace_back` expects the arguments of item constructor. In your case, it's a copy that isn't really required.

**Review by nest1ing — COMMENTED** [review]
As I wrote in TG, this functionality seems to me useless.

File: list-sensors.cpp (hunk 1 above)
Comment (alatarum) [reply list-sensors.cpp:415]: sorry, don't get your point here, using just
`sensors.emplace_back(service.first, obj.first);`
don't compiles.

**Review by AlexanderAmelkin — APPROVED** [review]
This may come in handy some day, so I'm merging this.

**Pull request merged**